lib/active_admin/base_controller.rb

```diff
@@ -59,6 +59,10 @@ def active_admin_namespace
     end
     helper_method :active_admin_namespace
 
+    def status_code
+      ActionDispatch::ExceptionWrapper.new(env, env["active_admin.original_error"]).status_code
```

Comment:
This method shouldn't placed in the BaseController or is there a specific reason for that?